### PR TITLE
chore: add proper labels

### LINKED ISSUE #114: Default and unique label sets that comply with the k8s specifications
All invenio k8s resources (deployment, service, etc) should have a unique label set for common and selector labels which can be assigned to a helm release and matches the k8s conventions.

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Possible solution:

These label sets are also generated by `helm create...` by default:

```yaml
{{/*
Common labels
*/}}
{{- define "invenio.labels" -}}
helm.sh/chart: {{ include "invenio.chart" . }}
{{ include "invenio.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}

{{/*
Selector labels
*/}}
{{- define "invenio.selectorLabels" -}}
app.kubernetes.io/name: {{ include "invenio.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
{{- end }}
```

### PR DESCRIPTION
**Pull request by egabancho**
* closes #114

### CODE REVIEW

File: charts/invenio/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+Selector labels
+*/}}
+{{- define "invenio.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "invenio.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```
Comment (lindhe) [inline charts/invenio/templates/_helpers.tpl:49]: I think it's valid to define a template after it's used (it's used on line 38 and defined here at line 46). But I think it reads better for people if we define it first. Also it's missing a blank newline.

File: charts/invenio/templates/_helpers.tpl [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+{{- define "invenio.labels" -}}
+helm.sh/chart: {{ include "invenio.chart" . }}
+{{ include "invenio.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+Selector labels
+*/}}
```
Comment (lindhe) [inline charts/invenio/templates/_helpers.tpl:45]: Missing start of block comment: `{{*/`

File: charts/invenio/templates/_helpers.tpl [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+{{- define "invenio.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "invenio.fullname" -}}
```
Comment (lindhe) [inline charts/invenio/templates/_helpers.tpl:13]: Seems like you define `invenio.fullname` but never use it.

File: charts/invenio/templates/_helpers.tpl (hunk 1 above)
Comment (egabancho) [reply charts/invenio/templates/_helpers.tpl:51]: I just ran `helm create invenio` on a different folder and copy over the content from `_helpers.tpl` ☺️

File: charts/invenio/templates/_helpers.tpl (hunk 3 above)
Comment (egabancho) [reply charts/invenio/templates/_helpers.tpl]: Removed.

**Review by lindhe — APPROVED** [review]

**Pull request merged**